README.md

```diff
@@ -1,70 +1,45 @@
-# Getting Started with Create React App
+# Covid-Tracker
+A Covid Tracker Webapp with a mobile-first design that displays the present number of confirmed cases of Covid across selected countries, incorporating the Narrative API to a React-Redux framework.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<!-- ![screenshot](src/assets/screenshot.png) -->
 
-## Available Scripts
+## Live Demo
 
-In the project directory, you can run:
+[Coming Soon on this address!](https://covid-tracker-microverse-project.netlify.app/) Check back often to see our project growing, or
+visit the [project's GitHub](https://github.com/iKuartz/covid-tracker) to look at its evolution commit by commit :)
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- Languages: HTML, CSS and JS
+- Tools: React, Redux, Bootstrap, Webpack, NPM, Jest
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Getting Started
 
-### `npm test`
+Clone
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```sh
+   git clone git@github.com:iKuartz/covid-tracker.git
+```
 
-### `npm run build`
+Run
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+```sh
+   cd covid-tracker
+   npm install
+   npm start
+```
+## Author
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### 👤 Ivan Silva
+- GitHub: [iKuartz](https://github.com/iKuartz)
+- LinkedIn: [Ivan Barbosa da Silva](https://www.linkedin.com/in/ivan-silva-a47058b3/)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Contributing
 
```

Comment:
Please add the `Acknowledgements` section or similar in README that gives appropriate credit to
the design author.

Reply:
Totally right, forgot that completely, already corrected.